srtcore/api.cpp

```diff
@@ -1087,6 +1582,7 @@ int CUDTUnited::close(CUDTSocket* s)
 
        // since "s" is located before m_GlobControlLock, locate it again in case
        // it became invalid
+       // XXX Seriously? Why not do all things necessary on s prior to deletion?
```

Comment:
Better to replace with guidance on improvement.

Reply:
Improved.